ciphey/iface/_modules.py

```diff
@@ -358,8 +361,11 @@ def add_one():
     ret = ret[:-1]
 
     # If we didn't show intermediate steps, then print the final result
-    if not display_intermediate:
+    if already_broken:
+        ret += f"""\nPlaintext: [bold green]"{escape(res.path[-1].result.value)}"[bold green]"""
+    else:
         ret += f"""Plaintext: [bold green]"{escape(res.path[-1].result.value)}"[bold green]"""
+
```

Comment:
This fixes the printing bug. I plan to extract into another module in another PR and clean up this.... mess....